config/locales/document_sets/document_sets-es.yml

```diff
@@ -19,14 +19,15 @@ es:
       url_description: Utilice letras minúsculas y guiones entre cualquier palabra.
     edit_privacy:
       document_set_collaborators: Colaboradores permitidos
-      document_set_privacy: 'Privacidad del Conjunto de Documentos: <i>%{privacy}</i>'
+      document_set_privacy_private: 'Conjunto de documentos Privacidad: <i>Privado</i>'
```

Comment:
Let's make this "Privacidad del Conjunto de Documentos" as in the old message -- same for following two.